websites_list/news_websites.csv

```diff
@@ -1988,3 +1988,13 @@
 1986,New Indian Express,newindianexpress.com,Right Leaning
 1987,Dainik Bhaskar,bhaskar.com,Pro-NDA
 1988,Dainik Jagran,jagran.com,Pro-NDA
+1989,moneycontrol,moneycontrol.com,Ecnonomic news. Owned by TV18
```

Comment:
control money, don't let money control you

Reply:
1821, Guardian, https://www.theguardian.com/international, Liberal Democrats

Reply:
1964, The Sun, https://www.thesun.co.uk/, Conservative Party

Reply:
1999, http://www.taipeitimes.com/, owned by The Liberty Times Group

Reply:
1971, The Star, https://www.thestar.com.my/, left leaning

Reply:
1999, Malaysiakini, https://www.malaysiakini.com/, claims to be pro-government

Reply:
1980, Liberty Times, https://www.ltn.com.tw/, centre-left

Reply:
1951, United Daily News, http://udn.com/NEWS/, centre-right

Reply:
2003, Apple Daily, https://tw.appledaily.com/, liberal

Reply:
1950, China Times, https://www.chinatimes.com/, centre-right

Reply:
1949, Taiwan News, https://www.taiwannews.com.tw/, centre-left

Reply:
1952, The China Post, https://chinapost.nownews.com/, centre-right

Reply:
1978, The Borneo Post, https://www.theborneopost.com/, daily english news in Sarawak and Sabah

Reply:
1976, Business Times, https://www.businesstimes.com.sg/, Owned by Singapore Press Holdings

Reply:
1994, The Edge, https://www.theedgemarkets.com/categories/malaysia, owned by The Edge Communications Sdn Bhd

Reply:
1949, Frankfurter Allgemeine Zeitung, https://www.faz.net/aktuell/, centre-right

Reply:
1854, Whitby Gazette, https://www.whitbygazette.co.uk/, impartial

Reply:
1996, Scottish Socialist Voice, https://socialistvoice.scot/, left leaning

Reply:
2009, The Tab, https://thetab.com/, left leaning

Reply:
1845, New Straits Times, https://www.nst.com.my/, Owned by Media Prima

Reply:
1946, China Press, https://www.chinapress.com.my/, owned by The China Press Berhad

Reply:
2006, Sinar Harian, https://www.sinarharian.com.my/, Malay language daily newspaper

Reply:
2007, L'essentiel, www.lessentiel.lu/fr/, free daily newspaper

Reply:
2000, Today, https://www.todayonline.com/, free daily newspaper

Reply:
1998, National Post, https://nationalpost.com/, liberal

Reply:
1980, Calgary Sun, https://calgarysun.com/, conservative

Reply:
1905, Lethbridge Herald, https://lethbridgeherald.com/, centre-left

Reply:
1936, The Globe and Mail, https://www.theglobeandmail.com/, centre-right

Reply:
1882, The Daily Courier https://www.dailycourier.com/, liberal

Reply:
1903, Edmonton Journal, https://edmontonjournal.com/, conservative